watertap/unit_models/crystallizer.py

```diff
@@ -664,6 +664,10 @@ def initialize(
         blk.properties_in.release_state(flags, outlvl=outlvl)
         init_log.info("Initialization Complete: {}".format(idaeslog.condition(res)))
 
+    def get_costing(self, module=None, **kwargs):
+        self.costing = Block()
+        module.Crystallization_costing(self.costing)
```

Comment:
I'm not sure this code is strictly necessary -- we do not currently use the "old" IDAES costing framework anywhere in WaterTAP, and we should probably consider removing this method from our other unit models.

Reply:
Done